addons/sys_list/fnc__showHintBox.sqf

```diff
@@ -1,23 +1,66 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Handles the ACRE notification system
  *
  * Arguments:
- * 0: ARGUMENT ONE <TYPE>
- * 1: ARGUMENT TWO <TYPE>
+ * 0: Display of CutRsc <DISPLAY>
+ * 1: Type (either flashing background or actual notification) <NUMBER>
+ * 2: Buffer pointer (defines at which position the notification is shown) <NUMBER>
  *
  * Return Value:
- * RETURN VALUE <TYPE>
+ * None
  *
  * Example:
- * [ARGUMENTS] call acre_COMPONENT_fnc_FUNCTIONNAME
+ * [(_this select 0), 0, acre_sys_list_hintBufferPointer] call acre_sys_list_fnc__showHintBox";
  *
  * Public: No
  */
 #include "script_component.hpp"
 
-params ["_hintIDD"];
+params ["_hintIDD", "_type", "_bufferpointer"];
 
-(_hintIDD displayCtrl 19100) ctrlSetText GVAR(hintTitle);
-(_hintIDD displayCtrl 19101) ctrlSetText GVAR(hintLine1);
-(_hintIDD displayCtrl 19102) ctrlSetText GVAR(hintLine2);
+if (_type == 0) then {
+    private _color = GVAR(hintColor);
+    _color set [3, 1];
+    (_hintIDD displayCtrl 19105) ctrlSetBackgroundColor _color;
```

Comment:
Substituting 19105 with a define would be better to maintain the code later on. Same for all other displayCtrl Ids

Reply:
Agree